### DISCUSSION TITLE
"My sites" screen: More data & sorting

### DISCUSSION THREAD

**Discussion by purplespider** [plausible/analytics#1035] [Anything Else]
As a freelance web developer, I'm looking to offer Plausible on all the websites I create. It would be great if the "My sites" screen could display some more stats about each site (currently it just shows "# visitors in last 24h"), such as:

- Total visitors in past 30 days
- Total page views in past 30 days
- Total "current" visitors (#777)

Maybe the ability to switch to a table view (instead of the existing grid). This would also make it possible to sort on the numbers to easily identify which website has had the most visitors. 

Here is how Matomo do it:
![image](https://user-images.githubusercontent.com/329880/118245584-ea11dd00-b498-11eb-8683-863cd4a11fa8.png)

**Comment by ukutaht** [comment]
Makes sense, thanks for the suggestion.

Something like this is planned for when we start using materialized views in our database. Currently it would be too expensive to calculate all of these metrics plus a chart for ~20 websites. Once we have some eager rollup tables, it will be trivial to add this information to the "sites" page.

One idea I've seen as well is adding a dropdown at the top so you can select the time frame.

**Comment by purplespider** [comment]
Just came across this "All sites" screen on another analytics tool: 🤤

![image](https://user-images.githubusercontent.com/329880/118787848-1f4c6f80-b88b-11eb-961e-48abf282a880.png)

**Reply by wheeleran** [reply]
that's a lovely looking dashboard, mind if i ask which other another it is?

**Comment by LauGau** [comment]
Any improvement on this?
I desperatly need to change the time frame for the "all sites" view...
Having only "for the last 24h" is really not enough... No need to be able granular on this I think, just being to display the number of visits for "the last 30days" would be greatly beneficial. 24h is really not representative for all my client websites (that are really small)...

**Comment by T-101** [comment]
It would be nice to have more than 18 sites listed in the landing page overview. I have 22 and counting, mildly inconvenient to click to second page to view status of all sites. I am using self-hosted version, if that makes any difference.

**Comment by gbissland** [comment]
Agreed! I think there was a feature request somewhere for adding more sites to the 'home' screen although I can't find it now. 
We are dealing with near 100 and it is tough to navigate.

There is a work around which was mentioned in a thread adding `?per_page=xx`    eg  `https://plausible.io/sites?per_page=100`
It still means you have to bookmark it.

**Reply by T-101** [reply]
ah, that url parameter works, thanks for the tip! still, wouldn't hurt to have a small option for this that get saved to localStorage :)

**Reply by T-101** [reply]
side note: this value seems to up for edit in the elixir file in the main repo /lib/plausible/repo.ex but I was unable to find it in the image that was pulled from docker hub

**Comment by metmarkosaric** [comment]
**Update**: You can now pin sites that you want to feature at the top of your sites page. You can pin up to nine sites. The most recently pinned site will automatically appear at the top of your list, helping you organize your dashboards by how often you use them.

Your pinned sites will also show up in the site switcher list that you can see within each dashboard by clicking on the site name in the top left. You can navigate between the sites in the site switcher list using keyboard shortcuts 1 to 9.

**Comment by wheeleran** [comment]
Am reviewing the doc about the [landing page](https://plausible.io/docs/landing-page) and there's a whole bunch of things that i just don't have. Site graph? nope. Gravatar? nope. Pin site? also nope. i'm self-hosting a docker image, on 2.0.0. Are the docs from the future?
